packages/core/src/components/Tooltip.svelte

```diff
@@ -97,22 +158,113 @@
   style="--stage-color: {activeMarkerInformation?.message.onboardingStage
     .backgroundColor}"
 >
-  <div class="visahoi-tooltip-title">
-    <b>{$activeMarker?.tooltip.title}</b>
-    <div class="visahoi-close-tooltip" on:click={closeTooltip}>
-      <i class="fas fa-times" />
+  <div class="visahoi-tooltip-header">
+    <div class="visahoi-tooltip-title">
+      {#if editTooltip}
+        <input class="visahoi-edit-title" type="text" bind:value={tempTitle} />
+      {:else}
+        <b>{$activeMarker?.tooltip.title}</b>
+      {/if}
+    </div>
+
+    <!--The trash icon is shown in the tooltip only isEditModeActive is set to true-->
+
+    <div class="visahoi-header-icons">
+      {#if $isEditModeActive && !editTooltip}
+        <div
+          class="visahoi-edit-tooltip"
```

Comment:
Please add a title attribute to all icons we have:
- [ ] close tooltip
- [ ] edit tooltip
- [ ] delete tooltip and marker
- [ ] cancel and do not save changes
- [ ] save changes